tests/func/experiments/test_remove.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+from funcy import first
+
+from dvc.exceptions import InvalidArgumentError
+from dvc.repo.experiments.utils import exp_refs_by_rev
+
+
+def test_remove_experiments_by_ref(tmp_dir, scm, dvc, exp_stage, caplog):
+    queue_length = 3
+    ref_list = []
+
+    for i in range(queue_length):
+        results = dvc.experiments.run(
+            exp_stage.addressing, params=[f"foo={i}"]
+        )
+        ref_info = first(exp_refs_by_rev(scm, first(results)))
+        ref_list.append(str(ref_info))
+
+    with pytest.raises(InvalidArgumentError):
+        assert dvc.experiments.remove(ref_list[:2] + ["non-exist"])
```

Comment:
Would `dvc.experiments.remove(["non-exist"] + ref_list[:2])` have the same behavior?

Reply:
of course.